openedx_events/__init__.py

```diff
@@ -5,4 +5,4 @@
 more information about the project.
 """
 
-__version__ = "9.5.2"
+__version__ = "9.5.3"
```

Comment:
This should be a minor bump since we're adding a new feature.

Reply:
Updated